website/cue/reference/components/sources/base/amqp.cue

```diff
@@ -309,6 +309,31 @@ base: components: sources: amqp: configuration: {
 					}
 				}
 			}
+			chunked_gelf: {
+				description:   "Options for the chunked gelf decoder."
+				relevant_when: "method = \"chunked_gelf\""
+				required:      false
+				type: object: options: {
+					pending_messages_limit: {
+						description: """
+																The maximum number of pending uncomplete messages. If this limit is reached, the decoder will start
+																dropping chunks of new messages. This limit ensures the memory usage of the decoder's state is bounded.
+																The default value is 1000.
+																"""
+						required: false
+						type: uint: default: 1000
```

Comment:
seeing that the default value is here... maybe we should remove it from the description field?

Reply:
Agreed, let's drop it from the description.